test/aws/scaleup.yml

```diff
@@ -32,6 +26,11 @@
       owner: root
       group: root
       mode: 0644
+  - name: Initialize openshift repos
+    import_role:
+      name: openshift_repos
+    vars:
+      openshift_version: "4.0"
```

Comment:
my 0.02$ - due to [variable precedence order](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable) wouldn't be better to move this variable into inventory file otherwise you'll need to change the code with a next 4.x version

Reply:
I agree, thanks. If we don't drop the role then we will sort this out later.